### ISSUE TITLE
Add option for rescheduling sessions

### ISSUE THREAD

**Issue by iamareebjamal** [fossasia/open-event-orga-app#173]
To be implemented after #172 

Design proposal ->

Click on a particular session should show a bottom sheet with the configurable fields using Standard Time Picker from Android,

**Comment by mariobehling** [comment]
This is out of scope of the app.

**Comment by iamareebjamal** [comment]
@mariobehling When @niranjan94 discussed the specification of the app in the meet, we discussed that these features were to be implemented:

`Planned features & enhancements are:
- Overview of sales
- Overview of tracks and sessions
- Quick session re-scheduling
- Push notifications for certain triggers`

Did the goal of app changed in somewhere between?
